pipeline/apiserv/main.go

```diff
@@ -242,6 +243,35 @@ func gatherAppsEndpoint(w http.ResponseWriter, r *http.Request) {
 
 }
 
+// altAppsEndpoint allows for external entities to query for alternative apps based on app ID.
+func altAppsEndpoint(w http.ResponseWriter, r *http.Request) {
+	mime := r.Header.Get("Accept")
+	if r.Method == "POST" || r.Method == "GET" {
+		if _, ok := supportedMimes[mime]; !ok {
```

Comment:
Still think at some point this should be a functional component in checking the method